Projects/Playground/Playground.Core/App.cs

```diff
@@ -27,7 +27,7 @@ public override void Initialize()
 
             Mvx.IoCProvider.RegisterSingleton<IMvxTextProvider>(new TextProviderBuilder().TextProvider);
 
-            RegisterAppStart<RootViewModel>();
+            // RegisterAppStart<RootViewModel>();
```

Comment:
This obviously won't work for all the other platforms.

Reply:
pls ignore this commit